packages/rs-dpp/src/validation/validation_result.rs

```diff
@@ -1,26 +1,101 @@
 use crate::errors::consensus::ConsensusError;
 use crate::ProtocolError;
+use std::fmt::Debug;
 
-pub type SimpleValidationResult = ValidationResult<()>;
+pub type SimpleConsensusValidationResult = ConsensusValidationResult<()>;
 
-#[derive(Default, Debug)]
-pub struct ValidationResult<TData: Clone> {
-    pub errors: Vec<ConsensusError>,
-    data: Option<TData>,
+pub type SimpleValidationResult<E> = ValidationResult<(), E>;
+
+pub type ConsensusValidationResult<TData> = ValidationResult<TData, ConsensusError>;
```

Comment:
Just a small nitpick, I think it's better to have them in a different order, i.e. SimpleValidationResult first, then ConsensusValidationResult, and then SimpleConsensusValidationResult, as it uses the previous type